README.md

```diff
@@ -52,9 +53,9 @@ run.config:
 ---
 
 #### postgresql_client_version
```

Comment:
One last minor quibble, and we'll be good to go! If you could wrap `postgresql_client_version` and `boxfile.yml` in code ticks, that'd be perfect.

Reply:
6ad046d288089cb7c91df945360f6382e608e56d